scripts/transform/apply_migration.py

```
@@ -0,0 +1,38 @@
+load_dotenv()
+
+def apply_migration(sql_file):
+    conn = psycopg2.connect(
+        host=os.getenv('PGHOST', 'localhost'),
+        port=os.getenv('PGPORT', '5432'),
+        database=os.getenv('PGDATABASE', 'retrosheet'),
+        user=os.getenv('PGUSER', os.getenv('USER')),
+        password=os.getenv('PGPASSWORD')
+    )
+    
+    with open(sql_file, 'r') as f:
```

Ruff (0.15.10)

[warning] Unnecessary mode argument

Remove mode argument

(UP015)